UKHomeOffice/sgmr-service
# SPCR-200 Swap to latest govuk-frontend package

## Description
We were not using the correct GDS Transport font that should be used on public facing services - we had an old version using Inter font.

To correct this need to move to the latest version of GDS frontend

- remove Inter font files
- upgrade to latest govuk-frontend package (gives us the correct font)
- remove old imports from scss files, not needed with this version
- remove overrides that was making styling work with the old package
- update items/styles that are not supported in new package defaults

## To test
- check font is `transport` and not `inter`
- check `add person` form looks correct (had to adjust hint text as it was running inline instead of block)
- check `save voyage` `exit without saving` text looks correct (had to adjust paragraph class as it was missing)
- click around and see if anything looks off

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
